modules/persistent-storage-csi-azure-disk-perf-plus-create-new-disk.adoc

```diff
@@ -0,0 +1,28 @@
+//
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-azure.adoc
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-azure-disk-perf-plus-create-new-disk_{context}"]
+= Enabling performance plus by creating new disks
+
+Normally, performance plus can be enabled only on new disks. For a workaround, you can use this procedure.
+
+.Prerequisites
+
+* Access to a Microsoft Azure cluster with cluster-admin privileges.
+
+* Access to an Azure disk with performance plus enabled.
+
```

Comment:
We don't need to access to Azure, we just "snapshot" or "clone" it by creating OCP object.

Reply:
I've revised it.